src/psyclone/tests/psyad/tl2ad_test.py

```diff
@@ -268,13 +270,13 @@ def test_generate_adjoint_str_generate_harness_logging(caplog):
         "end module my_mod\n"
     )
     with caplog.at_level(logging.INFO):
-        _ = generate_adjoint_str("", tl_code, ["field"], create_test=True)
+        _ = generate_adjoint_str(tl_code, ["field"], create_test=True)
```

Comment:
How was this working before?

Reply:
It wasn't. It should have been an xfail with a big X, but turned into an xfail with a small x and we didn't notice.

Reply:
Ah, the perils of x-fail...